code/ai/aiturret.cpp

```diff
@@ -2098,7 +2110,18 @@ void turret_swarm_fire_from_turret(turret_swarm_info *tsi)
 		}
 
 		// muzzle flash?
-		if (Weapon_info[tsi->weapon_class].muzzle_flash >= 0) {
+		if (Weapon_info[tsi->weapon_class].muzzle_effect.isValid()) {
+			//vec3d gun_world_pos;
+			//vm_vec_unrotate(&gun_world_pos, turret_pos, &Objects[OBJ_INDEX(objp)].orient);
+			//vm_vec_add2(&gun_world_pos, &Objects[OBJ_INDEX(objp)].pos);
```

Comment:
If you're sure this isn't necessary should probably remove these.  It does appear to be already in world coordinates so probably not necessary, but if you're unsure you should test it. I'm otherwise not a big fan of adding commented-out code.

Reply:
Oh! Whoops, yeah, that's not supposed to be there. Thank you for catching it.